app/src/test/java/org/mozilla/tv/firefox/pocket/PocketVideoRepoTest.kt

```diff
@@ -0,0 +1,49 @@
+package org.mozilla.tv.firefox.pocket
+
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.tv.firefox.helpers.ext.assertValues
+import org.mozilla.tv.firefox.utils.BuildConfigDerivables
+import org.mozilla.tv.firefox.utils.PreventLiveDataMainLooperCrashRule
+
+class PocketVideoRepoTest {
+
+    @get:Rule val rule = PreventLiveDataMainLooperCrashRule()
+
+    private lateinit var pocketVideoRepo: PocketVideoRepo
+
+    @Before
+    fun setup() {
+        initWithLanguage { true }
+    }
+
+    fun initWithLanguage(isEnglish: () -> Boolean) {
+        val pocketEndpoint = object : PocketEndpoint("1") {
+            override suspend fun getRecommendedVideos(): List<PocketViewModel.FeedItem.Video>? =
+                listOf(
+                    PocketViewModel.FeedItem.Video(
+                        1,
+                        "Some video",
+                        "some url",
+                        "thumbnail",
+                        1,
+                        "author"
+                    )
+                )
+        }
+
+        pocketVideoRepo = PocketVideoRepo(
+            pocketEndpoint = pocketEndpoint,
+            localeIsEnglish = isEnglish,
+            pocketFeedStateMachine = PocketFeedStateMachine(),
+            buildConfigDerivables = BuildConfigDerivables(isEnglish)
+        )
+    }
+
+    @Test
```

Comment:
I wonder if we should add a test for changing locales - it's not a common case, but it's the most likely place where something could break.

Reply:
Hm, I guess we need to decide how we're handling those cases.  To be honest, I've been assuming that users change locales very infrequently, so I've essentially been dropping them on the floor.  With this code, if a non-English user switches to English they won't see Pocket until they restart the app.

@athomasmoz how would you like us to handle this case?